editor/src/clj/editor/collection.clj

```diff
@@ -179,9 +178,9 @@
 
 (g/defnk produce-go-build-targets [_node-id build-error build-resource ddf-message resource-property-build-targets source-build-targets pose]
   ;; Create a build-target for the referenced or embedded game object. Also tag
-  ;; on :instance-data with the overrides for this instance. This will later be
-  ;; extracted and compiled into the Collection - the overrides do not end up in
-  ;; the resulting game object binary.
+  ;; on :game-object-instance-data with the overrides for this instance. This
+  ;; will later be extracted and compiled into the Collection - the overrides do
+  ;; not end up in the resulting game object binary.
```

Comment:
The key `:instance-data` was used for both game object instances and component instances, which was very confusing.